### ISSUE TITLE
Some toolbar icons not readable in dark mode

### ISSUE THREAD

**Issue by ryandesign** [AppleCommander/AppleCommander#124]
Some toolbar icons, like those used for the Hex Dump and Raw Dump buttons, are not readable in dark mode:

![syntax-highlighting-dark](https://github.com/AppleCommander/AppleCommander/assets/429209/8409bf40-0e20-4ec4-af64-b182172e0a0d)

Compare with light mode:

![syntax-highlighting-light](https://github.com/AppleCommander/AppleCommander/assets/429209/4eaf0f31-77c1-40cc-a383-d5d0ad75851c)

I see some attempts at antialiasing both the text and the circles but you'll need more than a 1-bit mask to make that work for the circles and you'll need a way to draw the text part of the icons in white instead of black when in dark mode. Maybe separate icons for dark and light mode. Or maybe remove the text from the icons completely.

**Comment by a2geek** [comment]
I just bumped the version of the windowing toolkit if you want to try it. There are automatic builds now, so the binaries are all packaged into a zip file here: https://github.com/AppleCommander/AppleCommander/actions/runs/6635737861 (look for "build-artifacts" and you'll get a copy of everything).

**Comment by ryandesign** [comment]
The icons don't look any different in that snapshot.

Are the icons provided by the toolkit? I assumed they were unique to AppleCommander and located somewhere in this repository.

**Comment by a2geek** [comment]
You're correct. They are here: https://github.com/AppleCommander/AppleCommander/tree/main/lib/ac-swt-common/src/main/resources/com/webcodepro/applecommander/ui/images

I am more than willing to replace them. I think I pulled these from Eclipse itself. Are you aware of any that would work better? (I think the size could be larger as well without difficulty.)
